src/backend/web/templates/hof.html

```diff
@@ -11,7 +11,7 @@
     <div class="col-xs-12">
       <h1 class="end_header"><i>FIRST</i> Hall of Fame</h1>
 
-      <blockquote><p><em>The Chairman's Award is the most prestigious award in
+      <blockquote><p><em>The Impact Award is the most prestigious award in
```

Comment:
```suggestion
      <blockquote><p><em>The <i>FIRST</i> Impact Award (formerly Chairman's Award) is the most prestigious award in
```

Reply:
Borrowing the verbiage from here https://www.firstinspires.org/resource-library/frc/first-impact-award-resources